release-please-config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "include-component-in-tag": false,
+  "include-v-in-tag": true,
+  "separate-pull-requests": false,
+  "bump-minor-pre-major": true,
+  "bump-patch-for-minor-pre-major": true,
+  "packages": {
+    "apps/vs-agent": {
+      "changelog-path": "CHANGELOG.md",
```

Comment:
I think this is not needed to explicitly set CHANGELOG.md when it is on the default location.